src/python/grpcio/commands.py

```diff
@@ -309,46 +238,3 @@ def _add_eggs_to_path(self):
     """Fetch install and test requirements"""
     self.distribution.fetch_build_eggs(self.distribution.install_requires)
     self.distribution.fetch_build_eggs(self.distribution.tests_require)
-
```

Comment:
I can't highlight the actual line, but it seems the TestLite command is still included above.  Is this intended?